src/stubs.ts

```diff
@@ -42,6 +43,17 @@ const shouldNotStub = (type: ConcreteComponent) => doNotStubComponents.has(type)
 export const addToDoNotStubComponents = (type: ConcreteComponent) =>
   doNotStubComponents.add(type)
 
+const removeSymbols = <T = ComponentPropsOptions>(props: T): T => {
```

Comment:
nit: this could be renamed to `stringifySymbols` as it is not removing them anymore?

Reply:
ooc: is the generic type `T` necessary here? As it is only called once, can't we use `ComponentPropsOptions` directly?

Reply:
Removed that